package.json

```diff
@@ -43,8 +43,8 @@
     "@hapi/hoek": "^10.0.1",
     "joi": "^17.7.0",
     "js-yaml": "^4.1.0",
-    "screwdriver-data-schema": "^23.5.1",
-    "screwdriver-config-parser": "^10.4.3"
+    "screwdriver-config-parser": "file:../config-parser/screwdriver-config-parser-10.0.0.tgz",
+    "screwdriver-data-schema": "file:../data-schema/screwdriver-data-schema-22.0.0.tgz"
```

Comment:
TODO: Update the version after https://github.com/screwdriver-cd/data-schema/pull/575 is merged